### PR TITLE
fix(docs): fix typescript doc issues

### PR DESCRIPTION
**Pull request by niudai**

### CODE REVIEW

File: docs/en/src/guide/functions/index.md [hunk 1]
```diff
@@ -75,6 +75,8 @@ If you have never developed a cloud function before, it is recommended to follow
 - [Send HTTP Request](/guide/functions/http-request)
 - [Retrieve POST Parameters](/guide/functions/post-params)
 - [Retrieve GET Parameters](/guide/functions/get-params)
+- [Using Typescript](/guide/functions/typescript)
```
Comment (adcentury) [inline docs/en/src/guide/functions/index.md:78]: Should be `TypeScript`, not `Typescript`

**Review by adcentury — CHANGES_REQUESTED** [review]

File: docs/zh-cn/src/guide/functions/index.md [hunk 2]
```diff
@@ -73,6 +73,9 @@
 - [发送 HTTP 请求](/guide/functions/http-request)
 - [获取 POST 参数](/guide/functions/post-params)
 - [获取 GET 参数](/guide/functions/get-params)
+- [使用 Typescript](/guide/functions/typescript)
```
Comment (adcentury) [inline docs/zh-cn/src/guide/functions/index.md:76]: Same typo here.

**Review by adcentury — CHANGES_REQUESTED** [review]

**Comment by niudai** [comment]
已改

**Review by adcentury — APPROVED** [review]
LGTM.

**Pull request merged**